src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@using EA.Prsd.Core.Web.Mvc.RazorHelpers
+@using EA.Weee.Core.Helpers
+@using EA.Weee.Web.RazorHelpers
+
+@model EA.Weee.Web.Areas.Aatf.ViewModels.SelectYourAatfViewModel
+@{
+    var title = "Which site would you like to manage evidence notes for?";
+    ViewBag.Title = title + " - GOV.UK";
+}
+
+<section id="main-content">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+
+            @Html.ActionLink("Back", "ChooseActivity", "Home", new { area = "Scheme", pcsId = Model.OrganisationId }, new { @class = "govuk-back-link weee-back-link" })
```

Comment:
chooseActivity seems to be in the Aatf area, judging by the src/EA.Weee.Web/Areas/Aatf/Views/ChooseSite/IndexNoAatf.cshtml path, is that area="Scheme" correct?

Reply:
I believe scheme is the correct area, yes

Reply:
okay